tests/framework/binding.h

```diff
@@ -1242,6 +1244,54 @@ inline VkBufferCreateInfo Buffer::CreateInfo(VkDeviceSize size, VkFlags usage, c
     return info;
 }
 
+class Tensor : public internal::NonDispHandle<VkTensorARM> {
+  public:
+    explicit Tensor();
+    explicit Tensor(const Device &dev, const bool is_copy_tensor = false);
+    explicit Tensor(const Device &dev, const VkTensorDescriptionARM &desc);
+    explicit Tensor(const Device &dev, const VkTensorCreateInfoARM &info);
+    ~Tensor() noexcept;
+    void destroy() noexcept;
+
+    // vkCreateTensor()
+    void init_no_mem(const Device &dev);
+
+    const VkMemoryRequirements2 &get_memory_reqs();
+    void bind_to_mem();
+    VkFormat format() const { return description_.format; }
+    uint32_t dimension_count() const { return description_.dimensionCount; }
+    VkTensorDescriptionARM &description() { return description_; }
+    const VkTensorDescriptionARM &description() const { return description_; }
+
+    VkTensorCreateInfoARM &create_info() { return create_info_; }
+    const VkTensorCreateInfoARM &create_info() const { return create_info_; }
+
+  private:
+    const Device *device_ = nullptr;
+    VkTensorCreateInfoARM create_info_;
+    VkTensorDescriptionARM description_;
+    VkTensorMemoryRequirementsInfoARM mem_req_info_;
+    VkMemoryRequirements2 mem_reqs_;
+    std::vector<int64_t> dimensions_;
+    std::vector<int64_t> strides_;
+    vkt::DeviceMemory memory_;
+};
+
+class TensorView : public internal::NonDispHandle<VkTensorViewARM> {
+  public:
+    TensorView() = default;
+    explicit TensorView(const Device &dev, const VkTensorViewCreateInfoARM &info);
+    ~TensorView() noexcept;
+    void destroy() noexcept;
+
+    // vkCreateTensorViewARM
+    void init(const Device &dev, const VkTensorViewCreateInfoARM &info);
```

Comment:
```suggestion
    void Init(const Device &dev, const VkTensorViewCreateInfoARM &info);
```

(we recently had an issue as something were `init` and some where `Init` so now they are all just `Init`

Reply:
what about destroy?

Reply:
ya, I need to do that as well, but need to do it everywhere, which i'll be doing soon, so just be ahead and make it `Destory()`

Reply:
I'll keep an eye on `NON_DISPATCHABLE_HANDLE_DTOR` and when you change it I'll do it, right now of course `Destroy` doesn't build.

Reply:
Thanks, changed.